config/splits/development/.htaccess

```diff
@@ -0,0 +1,24 @@
+# Deny all requests from Apache 2.4+.
+<IfModule mod_authz_core.c>
+  Require all denied
+</IfModule>
+
+# Deny all requests from Apache 2.0-2.2.
+<IfModule !mod_authz_core.c>
+  Deny from all
+</IfModule>
+
+# Turn off all options we don't need.
+Options -Indexes -ExecCGI -Includes -MultiViews
+
+# Set the catch-all handler to prevent scripts from being executed.
+SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
+<Files *>
+  # Override the handler again if we're run later in the evaluation list.
+  SetHandler Drupal_Security_Do_Not_Remove_See_SA_2013_003
+</Files>
+
+# If we know how to do it safely, disable the PHP engine entirely.
+<IfModule mod_php5.c>
+  php_flag engine off
+</IfModule>
```

Comment:
Why has this been added and what does it do?

Reply:
it was automatically generated into that folder. I'm looking into it

Reply:
Apparently it makes the split folder more protected and solves a git issue where if the split has been configured but doesn't have any files added to it, git won't deploy that empty folder. This can cause config import to fail. This is why the module adds it automatically now.

Reply:
Okay sound.